capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -125,6 +125,25 @@ class ClubPage extends Component {
       });
   }
 
+  handleMemberChange = (member, type) => {
+    if (type === 'add') {
+
+      const club = this.state.club;
+      club.memberIDs.push(member.id);
+      
+      this.setState({ members: [...this.state.members, member], club });
+    }
+    else if (type === 'remove') {
```

Comment:
nit: Can you move the `else if` to be on the same line as the final `}` above?

Reply:
Ok